core/src/main/scala/kafka/admin/AdminClient.scala

```diff
@@ -114,6 +116,13 @@ class AdminClient(val time: Time,
     listAllGroupsFlattened.filter(_.protocolType == ConsumerProtocol.PROTOCOL_TYPE)
   }
 
+  def listGroupOffsets(groupId: String): Map[TopicPartition, PartitionData] = {
+    val coordinator = findCoordinator(groupId)
+    val responseBody = send(coordinator, ApiKeys.OFFSET_FETCH, OffsetFetchRequest.allPartitionsOffsetFetchRequest(groupId))
+    val response = responseBody.asInstanceOf[OffsetFetchResponse]
+    response.responseData().asScala.toMap
```

Comment:
I was wondering about this also. Unfortunately, it seems we can't remove the per-partition error code since we do authorization on the topics separately.